scripts/build.sh

```diff
@@ -1,27 +1,28 @@
 #!/usr/bin/env bash
+PATH="$PATH:./node_modules/.bin"
```

Comment:
`./node_modules/.bin` should be prepend to $PATH to avoid system installed versions to take priority. It preserves to call wrong versions (especially with compilers like `jade` and `coffee` an introduced some bugs complicated to trace.